src/components/documentComponents/SubDocumentList.js

```diff
@@ -0,0 +1,50 @@
+import { push } from "../../utils.js";
+
+export default function SubDocumentList({ $target, initialState }) {
+  const $editorFooter = document.createElement("nav");
+  $editorFooter.className = "editor-bottom-nav";
+  $target.appendChild($editorFooter);
+
+  this.state = initialState;
+
+  this.setState = (nextState) => {
+    this.state = nextState;
+    this.render();
+  };
+
+  this.render = () => {
+    const { documents } = this.state;
+    $editorFooter.innerHTML = `
+    <ul class="subdocument-list">
+      ${documents
+        .map(
+          ({ id, title }) => `
+        <li data-id=${id} class="subdocument-link">
+        <i class="fa-regular fa-file"></i>
+          <span class="subdocument-link-title">${
+            (title ?? "제목 없음") || (title === "" && "제목 없음")
+          }</span>
+        </li>
+        `
+        )
+        .join("")}
+    </ul>
+        `;
+  };
+
+  $editorFooter.addEventListener("click", (event) => {
+    event.stopPropagation();
+
+    const { target } = event;
+    const $li = target.closest("li");
+
+    let { id } = $li.dataset;
+    id = parseInt(id);
+
+    if (!isNaN(id)) {
+      push(`${id}`);
+    }
```

Comment:
다음과 같은 방식은 어떠세요? `rawId` 이름이 매우 구리지만...
```suggestion
const rawId = $li.dataset && $li.dataset.id;
const id = parseInt(rawId, 10);
if (!rawId || isNaN(id)) return;

push(`${id}`);
```